docs/testrunner-toolkit/configuration/cypress.md

```diff
@@ -674,7 +674,6 @@ Provides details related to the Cypress test configuration that are relevant for
         env:
           hello: world
         testFiles: [ "**/*.spec.js" ]
-        headless: false
```

Comment:
I understand this is not a sub property of `config`, but since this example shows the whole `suites:` element, perhaps we should keep the headless setting in for a complete example?

Reply:
Hi @tianfeng92 -- I know some headless stuff got merged recently. Do you want to review this PR from Felix? Looks good to me, but I am not 100%.